paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -375,18 +404,20 @@ class CustomerSheet @Inject internal constructor(
                 ),
                 callback = callback,
                 statusBarColor = statusBarColor,
-                configuration = configuration,
             )
         }
 
         internal fun PaymentSelection?.toPaymentOptionSelection(
-            paymentOptionFactory: PaymentOptionFactory
+            paymentOptionFactory: PaymentOptionFactory,
+            canUseGooglePay: Boolean,
```

Comment:
This change has me a little scared we're missing  test coverage for it. I think ideally we'd do this before the API change.